src/test/run-pass/macros/syntax-extension-cfg.rs

```diff
@@ -1,6 +1,7 @@
 // run-pass
 // compile-flags: --cfg foo --cfg qux="foo"
 
+#[cfg(not(miri))] // Miri does not support panics
```

Comment:
it does support panics, it just doesn't support `should_panic` tests or `catch_panic`

Reply:
Oh okay, will push now

Reply:
@oli-obk Review most recent push, reverted the miri commit